src/models/base/FirestoreCollection.test.ts

```diff
@@ -19,10 +19,8 @@ const names = new (class extends FirestoreCollection<NameData> {
   converter = converter;
 })();
 
-beforeEach(async () =>
-  clearFirestoreData(app.options as { projectId: string })
-);
-afterAll(async () => app.delete());
```

Comment:
The issue before with lines like this is that something that creates a promise would be called but it'd never be waited on. So we need these functions to either be non-async so they can handle the promise themselves or `await` the promise in this `async` function.